src/Serilog.Sinks.Console/Sinks/SystemConsole/ConsoleSink.cs

```diff
@@ -41,10 +42,20 @@ static ConsoleSink()
             ConsoleTheme theme,
             ITextFormatter formatter,
             LogEventLevel? standardErrorFromLevel)
+            : this(theme, formatter, standardErrorFromLevel, _defaultSyncRoot)
+        {
+        }
+
+        public ConsoleSink(
+            ConsoleTheme theme,
+            ITextFormatter formatter,
+            LogEventLevel? standardErrorFromLevel,
+            object syncRoot)
```

Comment:
This class isn't `public`, so we can have just the one, four-parameter constructor, without breaking anyone.

Reply:
Fixed